src/module.c

```diff
@@ -1392,13 +1432,16 @@ int RM_StringAppendBuffer(RedisModuleCtx *ctx, RedisModuleString *str, const cha
 }
 
 /* --------------------------------------------------------------------------
- * Reply APIs
+ * ## Reply APIs
+ *
+ * These functions are used for sending replies to commands created using
+ * RM_CreateCommand().
```

Comment:
```suggestion
 * These functions are used for sending replies to the client.
```

Reply:
Oh, I meant RM_Call()... But it doesn't need to be mentioned.